src/ray/gcs/gcs_server/gcs_job_manager.cc

```diff
@@ -165,6 +182,8 @@ void GcsJobManager::HandleMarkJobFinished(rpc::MarkJobFinishedRequest request,
       });
   if (!status.ok()) {
     send_reply(status);
+  } else {
+    ++finished_jobs_count_;
```

Comment:
can we move this into `MarkJobAsFinished` when we erase `running_job_ids_[id]`?

Reply:
Updated